egs/wsj/s5/steps/nnet3/xconfig_to_configs.py

```diff
@@ -25,6 +25,10 @@ def get_args():
                         help='Filename of input xconfig file')
     parser.add_argument('--config-dir', required=True,
                         help='Directory to write config files and variables')
+    parser.add_argument('--nnet-edits', type=str, default=None,
+                        action=common_lib.NullstrToNoneAction,
+                        help="Edit network before getting nnet3-info "
```

Comment:
Change the help message to:
```
This option is useful in case the network you are creating
does not have an output node called 'output' (e.g. for
multilingual setups).  You can set this to an edit-string like:
'rename-node old-name=xxx new-name=output'
if node xxx plays the role of the output node in this network.
'
```
I am not super happy with this mechanism but we can keep it as-is for now.